profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -50,7 +50,7 @@ function _tmgmt_poetry_translator_access($op, $translator, $account) {
 function tmgmt_poetry_tmgmt_translator_plugin_info() {
   return array(
     'poetry' => array(
-      'label' => t('DGT Connector'),
+      'label' => t('Poetry Connector'),
```

Comment:
Are you sure about this? Everywhere else than the code, we mention DGT connector.

Reply:
Reverted.